### PR TITLE
feat(design): Add new colors into base palette [JOB-105102]

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Multiple teams have requested the addition of a new set of colors primarily for the purposes of data visualization and presentation of insights.

This PR adds the base values that will allow the teams to begin leveraging these colors - semantic values as needed will be applied in subsequent changes.

### Added

- Teal and purple base colors

### Changed

- values of `blue--900` in dark mode and the saturation of the `orange` range

## Testing

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/design/src/tokens/baseColor.tokens.json [hunk 1]
```diff
@@ -293,34 +293,98 @@
...
-          "$value": "hsl(33, 71%, 29%)"
+          "$value": "hsl(31, 100%, 27%)"
         },
         "800": {
-          "$value": "hsl(33, 72%, 20%)"
+          "$value": "hsl(31, 100%, 18%)"
         },
         "900": {
-          "$value": "hsl(33, 72%, 13%)"
+          "$value": "hsl(31, 90%, 12%)"
```
Comment (chris-at-jobber) [inline packages/design/src/tokens/baseColor.tokens.json:320]: Standardize the hue across the orange range and drive up saturation, particularly on the darker (500–900) steps to maintain more of an "orange" as long as possible

File: packages/design/src/tokens/dark.tokens.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
                     "$value": "hsl(197, 18%, 25%)"
                 },
                 "900": {
-                    "$value": "hsl(197, 15%, 18%)"
+                    "$value": "hsl(197, 15%, 16%)"
```
Comment (chris-at-jobber) [inline packages/design/src/tokens/dark.tokens.json:14]: Tamps down the `blue--900` lightness a pinch in dark mode to maintain an equivalent contrast ratio with the rest of the steps

File: packages/design/src/tokens/baseColor.tokens.json [hunk 3]
```diff
@@ -373,20 +437,20 @@
       }
     },
     "teal": {
-      "$value": "rgb(62, 173, 146)"
+      "$value": "{color.base.teal-.500}"
```
Comment (Aiden-Brine) [inline packages/design/src/tokens/baseColor.tokens.json:440]: Just a reminder to do the same as purple here

**Comment by Aiden-Brine** [comment]
This is looking good, just the one comment above and then I think you can put this up for review

**Comment by chris-at-jobber** [comment]
@Aiden-Brine pushed those purples up!

**Review by Aiden-Brine — APPROVED** [review]

**Pull request merged**